### PR TITLE
Document `# frozen_string_literal` in `bundle init` Gemfile

### PR DESCRIPTION
**Pull request by 315tky**
Thanks so much for the contribution!
To make reviewing this PR a bit easier, please fill out answers to the following questions.

### What was the end-user problem that led to this PR?

An enhancement request to bundle-init docu as per - issue 6140 

### What was your diagnosis of the problem?

n/a

### What is your fix for the problem, implemented in this PR?

Updated the man/bundle-init.ronn file to include brief description of frozen string literal, and added a SEE ALSO section with link to Gemfile(5) webpage

### Why did you choose this fix out of the possible options?

File edited to try and reflect the previous discussion regarding what was required.

### CODE REVIEW

File: man/bundle-init.ronn [hunk 1]
```diff
@@ -12,7 +12,13 @@ adding a [`Gemfile(5)`][Gemfile(5)] to a gem with a gemspec, the `--gemspec` opt
 automatically add each dependency listed in the gemspec file to the newly
 created [`Gemfile(5)`][Gemfile(5)].
 
+One of the lines included in the default [`Gemfile(5)`][Gemfile(5)] generated is `# frozen_string_literal: true`. This is a magic comment supported for the first time in Ruby 2.3. The presence of this line results in all string literals in the file being implicitly frozen. Frozen string literals will be the default in Ruby 3.
```
Comment (colby-swandale) [inline man/bundle-init.ronn]: You need to split this line up to match the max width, which is about 80 characters. Checkout the other man pages to see what i'm talking about.

**Comment by 315tky** [comment]
Thanks, have corrected

File: man/bundle-init.ronn (hunk 1 above)
Comment (315tky) [reply man/bundle-init.ronn]: Thanks, have corrected

**Comment by rubymorillo** [comment]
@colby-swandale Hey Colby, do you think we can get this one merged? Thanks!

File: man/bundle-init.ronn [hunk 2]
```diff
@@ -12,7 +12,17 @@ adding a [`Gemfile(5)`][Gemfile(5)] to a gem with a gemspec, the `--gemspec` opt
 automatically add each dependency listed in the gemspec file to the newly
 created [`Gemfile(5)`][Gemfile(5)].
 
+One of the lines included in the default [`Gemfile(5)`][Gemfile(5)]
```
Comment (colby-swandale) [inline man/bundle-init.ronn]: This should be in it's own section as we're no talking longer about the command itself.

Comment (colby-swandale) [inline man/bundle-init.ronn]: > in the default

This implies there's a way to have a Gemfile without the magic comment, which there isn't. How about `included in the generated Gemfile is...`?

File: man/bundle-init.ronn [hunk 3]
```diff
@@ -16,3 +16,15 @@ created [`Gemfile(5)`][Gemfile(5)].
...
 * `--gemspec`:
   Use the specified .gemspec to create the [`Gemfile(5)`][Gemfile(5)]
+
+## FILES
+
+Included in the default [`Gemfile(5)`][Gemfile(5)]
+generated is the line `# frozen_string_literal: true`. This is a magic comment
+supported for the first time in Ruby 2.3. The presence of this line
+results in all string literals in the file being implicitly frozen.
+Frozen string literals will be the default in Ruby 3.
```
Comment (koic) [inline man/bundle-init.ronn]: AFAIK, Frozen string literal will be default in Ruby 3.0 or not yet determinable. Here `may be` seems more appropriate than `will be`.

**Comment by indirect** [comment]
@bundlerbot r+

**Pull request merged**